app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -2,20 +2,29 @@ import React, { useState } from "react";
 import { Modal } from "neetoui";
 import notesApi from "apis/notes";
 
-export default function DeleteAlert({ refetch, onClose, selectedNoteIds }) {
+export default function DeleteAlert({
```

Comment:
Instead of using a custom Modal component, please use neetoUI [Alert Component](https://neeto-ui.netlify.app/#/alert)